frontend/package.json

```diff
@@ -5,13 +5,15 @@
   "scripts": {
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
-    "lint": "vue-cli-service lint"
+    "lint": "vue-cli-service lint",
+    "http:server": "http-server dist -c-1"
   },
   "dependencies": {
+    "@7ninjas/scss-mixins": "^1.0.1",
```

Comment:
Can you please tell me why we need these mixins? It's a super-simple website and we should be able to create a consistent look with just bootstrap and css.

Of course, the explanation "because it's awesome and THIS AND THAT AND CHECKOUT THIS COOL FEATURE BRO!" is also a valid explanation ;)